src/Microsoft.AspNetCore.Diagnostics.Abstractions/IExceptionHandlerPathFeature.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Diagnostics
+{
+    public interface IExceptionHandlerPathFeature : IExceptionHandlerFeature
+    {
+        string Path { get; }
```

Comment:
Would it make sense to make this a Context type? Just so that we don't have to rev the interface any time we want to change it?